### ISSUE TITLE
Can't compile on Ubuntu 16.10

### ISSUE THREAD

**Issue by calopter** [rorywalsh/cabbage_v1_old#21]
Hi! I'm getting the following error compiling on Ubuntu 16.10 64bit. I'm using the VST SDK 3.6.6 because I can't find the 2.4 version anywhere.

`In file included from ../../JuceLibraryCode/modules/juce_audio_processors/juce_audio_processors.cpp:152:0:
../../JuceLibraryCode/modules/juce_audio_processors/format_types/juce_VSTPluginFormat.cpp:49:44: fatal error: pluginterfaces/vst2.x/aeffectx.h: No such file or directory`

**Comment by cabbageaudio** [comment]
The 2.4 sdk is included with the newer sdk, are you passing the path to the
vst sdk when you build?

**Comment by calopter** [comment]
I am. Just double checked the path, still failing

**Comment by rorywalsh** [comment]
Can you post the build command you are using, and the location of the SDK? Btw, have you looked at 
https://github.com/rorywalsh/cabbage/issues/20
It contains info on how to correctly pass the VST3 SDK path to the build script. Maybe it can help.

**Comment by calopter** [comment]
Ok, so I've since switched to the dev branch which seems to solve the sdk error (bit of a git noob, I haven't encountered a master branch that doesn't build...). Now I get the following error however. A bit of googling suggests it's some upstream problem with juce. I did as the forum poster suggested and it built. Thanks!

https://forum.juce.com/t/compilation-problem-on-fedora-24/18215/2
http://pastebin.com/vuxqN3w3
